server/commands/hubs.py

```diff
@@ -575,10 +576,38 @@ def ooc_cmd_unhide_clients(client, arg):
     )
 
 
+@mod_only(hub_owners=True)
+def ooc_cmd_force_follow(client, arg):
+    """
+    Force someone to follow you. Follow me!
+    Usage: /force_follow <id>
+    """
+    if len(arg) == 0:
+        raise ArgumentError("You must specify a target. Use /force_follow <id>.")
+    try:
+        targets = client.server.client_manager.get_targets(
+            client, TargetType.ID, int(arg), False
+        )
+    except:
+        raise ArgumentError("You must specify a target. Use /force_follow <id>.")
+    if targets:
+        for c in targets:
+            if client == c:
+                raise ClientError("You are already forced to follow yourself because you are yourself!")
+            c.following = client
+            c.forced_to_follow = True
+            c.send_ooc(f"You've been forced to follow {client.showname}!")
+            if c.area != client.area:
+                c.set_area(client.area)
+        client.send_ooc(f"Forced {len(targets)} existing client(s) to follow you.")
+    else:
+        client.send_ooc("No targets found.")
+
+
 def ooc_cmd_follow(client, arg):
     """
     Follow targeted character ID.
-    Usage: /follow [id]
+    Usage: /follow <id>
```

Comment:
[id] is not mandatory, you can use /follow by itself to check if you're following anyone or not

Reply:
oh sorry, I thought that was just a preference thing